app/Resources/views/collection.html.twig

```diff
@@ -4,10 +4,6 @@
 
 {% block title %}{{ title|striptags }} | Collections{% endblock %}
 
-{% block description %}
```

Comment:
doesn't look related (but fine if intended)

Reply:
The description can now be derived at the higher level (`page.html.twig`) as there's a consistent argument (`item`).